src/test/java/apollo/commands/InvalidCommandTest.java

```diff
@@ -0,0 +1,15 @@
+package apollo.commands;
+
+import apollo.exceptions.ApolloOutOfBoundsException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class InvalidCommandTest {
+    @Test
+    void execute() throws ApolloOutOfBoundsException {
+        Command invalid = new InvalidCommand();
+        String expected = "Apologies, I do not understand that. ";
+        assertEquals(expected, invalid.execute());
```

Comment:
Would it be nice to also print a list of keywords that Apollo recognizes on top of the apology? 
_It might be helpful for new users of Apollo who would be more prone to making such input mistakes 😄_